### DISCUSSION TITLE
【任务】【黑客松】灵魂绑定代币SBT系统 ｜100,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#170] [General]
目前WTF Solidity极简教程有45讲，也有很多人用这个教程学习。在路线图中，如果WTFSolidity repo超过1024个star，就会给学习过的人发课程认证（SoulBound Token）。其实SBT系统我们已经做了几个月了，已经有了雏形：[Soulbound Protocol](https://soulbound.life)。现在我们需要完善它的第一个产品 Soulbound POAP，它是POAP的升级版，无审查且不可转移。

这个项目会参加万向区块链举办的夏季黑客松（9月开始）。

<img width="1355" alt="截屏2022-08-15 下午7 29 53" src="https://user-images.githubusercontent.com/14728591/184627536-389749f1-f7f5-4c04-8f12-4b54ebde2192.png">

**任务描述**：完成[Soulbound Protocol](https://soulbound.life)的第一个产品sPOAP的MVP（Minimal Viable Product）。包括但不限于以下内容：
1. 完善soulbound.life前端
2. sPOAP（Soulbound POAP）合约的完善，测试，部署：https://github.com/SoulBoundProtocol/sPOAP
3. sPOAP后端，包括一份外部组织可以调用的sdk，方便DAO组织创建、发放sPOAP。
4. UI设计

**赏金**：100,000 $PEOPLE 代币 + 黑客松奖金，根据工作量分配，其中50,000 $PEOPLE会追溯给8月前soulbound protocol的贡献者（默认参与这个任务）。

Soulbound OG贡献者：@0xhay @apexglory @bearfishx @djia @Glacier-Luo @Rookie93118 @zerokarafont\

请有意参加的WTF贡献者/成员回帖报名，并介绍下自己的技术栈（例如 前端：html/css/js, react）！另外如果还没填过WTF社区申请表的，也填一下：[表格](https://docs.google.com/spreadsheets/d/13eMBECDK4YQXhmFz0j8FPpEthUKYmTlaxILbIHnm1iw/edit?resourcekey#gid=507424848)

**Comment by fox493** [comment]
前端vue nuxt
后端 node

**Reply by AmazingAng** [reply]
你微信多少？

**Reply by fox493** [reply]
foxof_eth

**Comment by hyc6446** [comment]
合约的完善测试需要人员吗？想参与进来

**Reply by AmazingAng** [reply]
需要的

**Reply by thurendous** [reply]
想要参加测试

**Reply by hyc6446** [reply]
申请合约完善
前端：html+css(less/sass)+js(ts)+vue
后端：python(golang开发时间不久)
合约：solidity

**Reply by AmazingAng** [reply]
> 申请合约完善

可以的。你微信多少

**Reply by AmazingAng** [reply]
> hyc6446

你微信加不了

**Comment by EasyChris** [comment]
前端：vue 
后端：python nodejs 还会一点solidity

**Reply by AmazingAng** [reply]
好

**Comment by FlyingShuriken** [comment]
前端: ReactJS (TS>JS), Jinja2 (Python)
后端: Python (Flask), NextJS
会一点solidity

**Reply by AmazingAng** [reply]
好 我wx跟你说

**Reply by FlyingShuriken** [reply]
好的

**Comment by Doctor-DC** [comment]
前端: react vue

**Comment by boombb12138** [comment]
前端：vue+typescript
UI设计

**Reply by AmazingAng** [reply]
你的微信多少？

**Reply by boombb12138** [reply]
18218239073

**Comment by PigCharid** [comment]
Golang后端+solidity 了解过类似项目mycryptoprofile

**Reply by AmazingAng** [reply]
你wx多少？

**Reply by PigCharid** [reply]
13721072141

**Comment by zengh-cropty** [comment]
java后端+solidity+前端js没问题

**Reply by zengh-cropty** [reply]
wx:zenghang184390
